pkg/bff/announcements.go

```diff
@@ -21,6 +21,14 @@ const (
 	subMonths        = -2
 )
 
+// @Summary Get announcements [read:announcements]
```

Comment:
Unfortunately there wasn't a nicer way to specify the permission scopes in the annotation so my solution was to put them into the summary. It might be possible to specify a custom security configuration but it would take some more research.